src/components/StaticContent/Footer.re

```diff
@@ -74,4 +81,4 @@ module Footer = {
         </div>
       </div>
     </Box>;
-};
+};
```

Comment:
Could you set your vs code to have: `"files.insertFinalNewline": true` in the settings?